server/channels/app/platform/web_conn.go

```diff
@@ -232,9 +232,11 @@ func (ps *PlatformService) NewWebConn(cfg *WebConnConfig, suite SuiteIFace, runn
 	wc.SetSessionExpiresAt(cfg.Session.ExpiresAt)
 	wc.SetConnectionID(cfg.ConnectionID)
 
-	wc.Platform.Go(func() {
-		wc.HookRunner.RunMultiHook(func(hooks plugin.Hooks) bool {
-			hooks.OnWebSocketConnect(wc.GetConnectionID(), wc.UserId)
+	connID := wc.GetConnectionID()
```

Comment:
connectionID is already an atomic, so I don't think we need that. The race is only happening on accessing `wc.UserId`.

However, `wc.UserId` is being accessed in a lot of other `platform.Go` methods as well. Let's change those as well for completeness.

Reply:
Makes sense 👍